File: Coding_Platforms/Codewars/Golang/spinning_words.go
package main

import (
	"fmt"
	"strings"
)

func main() {

	fmt.Println(SpinWords("i love pizza and burger"))
}

func SpinWords(str string) string { //spinning words

	//create two
	word_list := strings.Split(str, " ")
	result_list := []string{}

	for _, word := range word_list {

		if len(word) > 4 {

			reverse_word := ""
			for i := len(word); i > 0; i-- {

				reverse_word += string(word[i-1])
			}
			fmt.Println(reverse_word)

			result_list = append(result_list, reverse_word)
		} else {
			result_list = append(result_list, word)
		}

	}

	return strings.Join(result_list, " ")
}

File: Coding_Platforms/Codewars/Python/spinning_words.py
def spin_words(sentence):

    #create list for store words from sentence and
    #one veriable for index 
    word_list = [word for word in sentence.split(" ")]
    idx  = 0

    for word in word_list: # iterate over the list

        if len(word) >4: # if word lenght is greater then four then reverse the word

            word_list[idx] = word[::-1]

        idx +=1

    # join the word of word_list by space
    return " ".join(word_list)


print(spin_words("i love pizza and burger"))